src/app/tabs/compile-tab.js

```diff
@@ -95,12 +96,36 @@ module.exports = class CompileTab {
           self.data.contractsDetails[contract.name] = parseContracts(contract.name, contract.object, self._opts.compiler.getSource(contract.file))
           var contractName = yo`<option>${contract.name}</option>`
           self._view.contractNames.appendChild(contractName)
+          for (let func in contract.object.evm.methodIdentifiers) {
+            var signature = contract.object.evm.methodIdentifiers[func]
+            var url = 'https://raw.githubusercontent.com/ethereum-lists/4bytes/master/signatures/' + signature
```

Comment:
@ligi  correct me if im wrong, but i think the endpoint should be 
https://www.4byte.directory/api/v1/signatures/?hex_signature=94f6e8ed

the result is something like
```
{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 80538,
            "created_at": "2018-06-28T01:05:14.381003Z",
            "text_signature": "initBallotProxy(uint8,bytes32,bytes32,bytes32[4])",
            "hex_signature": "0x94f6e8ed",
            "bytes_signature": "öèí"
        }
    ]
}
```
the warning should be displayed
if `count > 1`
if `count === 1 && the local text signature is different from the remote one`

Reply:
Ook, I got it, the actual code does not check if there is more than one function, updating it now!

Reply:
it's a pleasure =)
I don't know which is the correct file to put this call, so I put the verification inside contractParser.js and storing the collisions inside detail.collisions, the compile-tab.js will only display the warnings.